fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -29,17 +34,39 @@ class ConditionalSelectionRate(ClassificationMoment):
 
     """
 
+    def __init__(self, ratio=1.0):
+        """Intialise with the ratio value."""
+        super(ConditionalSelectionRate, self).__init__()
+        if ratio <= 0 or ratio > 1:
+            raise ValueError(_MESSAGE_RATIO_NOT_IN_RANGE)
+        self.ratio = ratio
+
     def default_objective(self):
         """Return the default objective for moments of this kind."""
         return ErrorRate()
 
-    def load_data(self, X, y, event=None, **kwargs):
+    def load_data(self, X, y, event=None, utility=None, **kwargs):
         """Load the specified data into this object.
 
         This adds a column `event` to the `tags` field.
+
+        The `utility` is a matrix which correspond to g(X,A,Y,h(X)) as mentioned
+        in the paper `Agarwal et al. (2018) <https://arxiv.org/abs/1803.02453>`
+        The `utility` defaults to h(X), ie. [1 in R^n, 0 in R^n]
```

Comment:
I'm unsure what `1 in R^n` means. Are you trying to say that the utility is [1, 0] for every sample?

Also, nit: i.e. instead of ie.

Reply:
Yes. I wanted to say that it has n rows (for every sample).